.github/actions/python-environment/action.yml

```diff
@@ -52,25 +52,28 @@ runs:
 
     - name: Setup cache variables
       id: setup-cache-variables
+      working-directory: ${{ inputs.working-directory }}
```

Comment:
Oh, does it mean until now the action took the wrong poetry.lock file?
Will we need to adapt all clients of the action to set the working directory?